python/feathub/online_stores/memory_online_store.py

```diff
@@ -138,3 +175,13 @@ def get(
         if table_info.timestamp_field is not None and not include_timestamp_field:
             features = features.drop(columns=[field_to_drop])
         return features
+
+    def reset(self) -> None:
```

Comment:
Do you think it would be better if we add `open()` & `close()`, or `initialize()` and `cleanup()` method to `FeathubClient` or `FeatureService` class? These methods would help users setup resources beforehand and closes them when they are sure they do not need them anymore. And we can invoke `MemoryOnlineStore.reset()` in those methods.

If not, this method is currently only used in tests. It might be better to create a utility class for this function in test_utils, instead of making it a public API.

Reply:
MemoryOnlineStore is only needed in tests. We agree it is simpler not to add those methods in FeathubClient or FeatureService.

RedisClient can be closed when FeatHubClient is destructed.